Lombiq.ContentEditors.Samples/Lombiq.ContentEditors.Samples.csproj

```diff
@@ -1,31 +1,48 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
-    <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
-        <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
-        <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**;Tests\**</DefaultItemExcludes>
-    </PropertyGroup>
-
-    <Import Project="..\..\..\Utilities\Lombiq.Npm.Targets\Lombiq.Npm.Targets.props" />
-    <Import Project="..\..\..\Utilities\Lombiq.Npm.Targets\Lombiq.Npm.Targets.targets" />
-    <Import Project="..\..\..\Utilities\Lombiq.Gulp.Extensions\Lombiq.Gulp.Extensions.targets" />
-
-    <ItemGroup>
-        <None Remove="node_modules\**" />
-        <None Remove="Tests\**" />
-    </ItemGroup>
-
-    <ItemGroup>
-        <FrameworkReference Include="Microsoft.AspNetCore.App" />
-    </ItemGroup>
-
-    <ItemGroup>
-        <PackageReference Include="OrchardCore.Module.Targets" Version="1.3.0" />
-        <PackageReference Include="OrchardCore.Contents" Version="1.3.0" />
-    </ItemGroup>
-
-    <ItemGroup>
-      <ProjectReference Include="..\Lombiq.ContentEditors\Lombiq.ContentEditors.csproj" />
-    </ItemGroup>
+  <PropertyGroup>

```

Comment:
So the README and the README for the Sample project (and other stuff needed for open sourcing) will be done in https://lombiq.atlassian.net/browse/OSOE-56 right?

Reply:
Yes, that's correct. For now the goal is to have it in a separate repository and have it somewhat functional.